masterlist.yaml

```diff
@@ -12959,6 +12961,9 @@ plugins:
           - 'Unique Landscapes - Merged ESP'
           - '[Unique Landscapes Compatibility Patches](https://www.nexusmods.com/oblivion/mods/13834/)'
         condition: 'version("Unique Landscapes.esp", "2.0.0", >=) and active("Unique Landscapes.esp") and not file("ULE KvatchRebuilt-CheydinhalFalls patch.esp")'
+      - <<: *patchIncluded
+        subs: [ 'Weather - All Natural' ]
+        condition: 'active("All Natural Base.esm") and not active("Kvatch Rebuilt Weather Patch.esp")'
```

Comment:
```suggestion
```

The text of `patchIncluded` is "You seem to be using **%1%**, but you have not enabled a compatibility patch for this mod. A compatibility patch is included with this plugin''s installer." This patch isn't included in _Kvatch Rebuilt_'s installer, but rather _Weather - All Natural_'s installer, so the message should be applied to it instead. You can do so in this PR if you want.